code/cutout.py

```diff
@@ -0,0 +1,115 @@
+# function to extract cutout image
+import math
+from astropy.nddata import Cutout2D
+import astropy.io.fits as fits
+from astropy.wcs import WCS
+
+from exceptions import CutoutError
+
+
+def extract_pair(ra, dec, *, img_pair, cutout_width, mosaic_pix_scale):
+    """Extract cutouts from the science and background images in `img_pair`.
+
+    If `img_pair` contains only one element, extract the cutout once and return it for both.
+
+    Parameters:
+    -----------
+    ra, dec: float or double
```

Comment:
I agree, `SkyCoords` and one arg rather than two would be great. I actually thought about doing that, but many functions would need both their signatures and internal code updated to handle it, so I didn't touch it here. Happy to make that change, but would prefer to do it in a separate PR.

Reply:
OK, a separate PR sounds good. I feel on the user level of the notebooks it's actually nice to force them to work with SkyCoords. (and one other option we do often in astroquery, but I don't necessarily say it's a good approach for this case, is to allow things that can a SkyCoord constructor consume. Here I would say that keeping it simple, and strict for skycoords is good)

Reply:
I like that approach. Both because it makes the code simpler and because it forces the user to be clear about what they're submitting (like units) so there's no room for confusion.